tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -107,15 +107,17 @@ def _table_has_clustering(self, table_obj):
     @mock.patch("gcloud.gcs.LOOKUP_TABLES_DATASET_ID", dataset_id)
     def test_all_files_unparseable_output(self):
         # TODO possible bug: if no pre-existing table, results in bq table not found error
-        for cdm_table in common.SUBMISSION_FILES:
+        for cdm_table in common.SUBMISSION_CSV_FILES:
```

Comment:
Are separate integration tests needed for jsonl and parquet files?

Reply:
There's an all encompassing five_person test that has all three file types. It is a full integration test so separate ones are not needed